### PR TITLE
Create monthly.py

### PR DESCRIPTION
**Pull request by pythexcel**

### CODE REVIEW

File: app/api/monthly.py [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+from flask_jwt_extended import (
+    JWTManager, jwt_required, create_access_token,
+    get_jwt_identity, get_current_user, jwt_refresh_token_required,
+    verify_jwt_in_request
+)
+from app.config import slack_token
+
+bp = Blueprint('monthly', __name__, url_prefix='/')
+
+# function for getting all the juniors of managers
```
Comment (reactexcel) [inline app/api/monthly.py]: this function need to be common. we cannot write this function everytime we need it. it used in 360 also. make it common in app.util

File: app/api/monthly.py [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+            "user": str(current_user["_id"]),
+            "type": "monthly",
+            "month": month
+        })
+        report = [add_user_data(serialize_doc(doc)) for doc in report]
+        return jsonify(report)
+    else:
+        #checking if join date is less than 7 or not if not subtract 7 from it
+        if datee > 7:
+            join_date = datee - 7
```
Comment (reactexcel) [inline app/api/monthly.py]: this was supposed to 3 to 4 days. not 7 days

File: app/api/monthly.py [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+        else:
+            join_date = datee
+
```
Comment (reactexcel) [inline app/api/monthly.py]: we also need a condition that atleast 3 weekly reports has been submitted by a user before a montly report is generated. or else even new joiners will start getting this

File: app/api/monthly.py (hunk 1 above)
Comment (pythexcel) [reply app/api/monthly.py]: sir in 360 he is finding managers of a particular user here i am finding juniors of a particular user they both are diffrent

File: app/api/monthly.py (hunk 2 above)
Comment (pythexcel) [reply app/api/monthly.py]: sir i am subtraction 3  days for  the  schdulers notification this condition is for when you can make monthly report before 7 days of date of joining ,if you want i will change it to 3 days

File: app/api/monthly.py (hunk 3 above)
Comment (pythexcel) [reply app/api/monthly.py]: working on this

File: app/api/monthly.py (hunk 2 above)
Comment (reactexcel) [reply app/api/monthly.py]: ok its fine

**Pull request merged**